cdap-api/src/main/java/co/cask/cdap/api/schedule/SchedulableProgramType.java

```diff
@@ -20,5 +20,5 @@
  * Defines types of programs that can be scheduled by the Application.
  */
 public enum SchedulableProgramType {
-  WORKFLOW, MAPREDUCE, SPARK, CUSTOM_ACTION
+  WORKFLOW, MAPREDUCE, SPARK, CUSTOM_ACTION, NONE
```

Comment:
`WorkflowSpecification` now have list of `WorkflowNode` instead of list of `ScheduleProgramInfo`. Each `WorkflowNode` in the list has `WorkflowNodeType` to identify type of the node (`ACTION`, `FORK`, `CONDITION`) and the `ScheduleProgramInfo` to identify the program associated with the `WorkflowNode`. In case of `ACTION` nodes, there will be some type of program associated with the WorkflowNode, however `FORK` and `CONDITION` type of WorkflowNode have no program associated with it, which is represented by `NONE` in the above enum. May be we can change the name of the enum to something similar to `WorkflowNodeProgramType` to make it more clear?